specs/agents/mobile/events.md

```diff
@@ -1,9 +1,12 @@
 ## Mobile Events
 
+Status: [Experimental](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/document-status.md)
+
 This document describes event used by the Mobile SDKs using the [OpenTelementry Event Api](https://github.com/open-telemetry/opentelemetry-specification/blob/0a4c6656d1ac1261cfe426b964fd63b1c302877d/specification/logs/event-api.md).
 All events collected by the mobile agents should set the `event.domain` to `device`.
-Event names will be recording using the `event.name`
+Event names will be recording using the `event.name`.
 
+Due to the dependency on Open Telemetry's events API this document's contents are subject to change and considered [experimental](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/document-status.md).
```

Comment:
nit: I see that they put "OpenTelemetry", the 2 words together on their website, so it's probably better to keep it that way over our docs too.

```suggestion
Due to the dependency on OpenTelemetry's events API this document's contents are subject to change and considered [experimental](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/document-status.md).
```